bundle/metadata/annotations.yaml

```
@@ -9,6 +9,7 @@ annotations:
   operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
   operators.operatorframework.io.metrics.builder: operator-sdk-v2.0.0+git
   operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v3
+  com.redhat.openshift.versions: "v4.11"
```

```suggestion
  com.redhat.openshift.versions: "=v4.11"
```
Per [link](https://redhat-connect.gitbook.io/certified-operator-guide/ocp-deployment/operator-metadata/bundle-directory/managing-openshift-versions).